README.rst

```diff
@@ -57,6 +57,8 @@ Go to the 'Site Setup' page in a Plone site and click on the 'Add-ons' link.
 
 Check the box next to ``collective.lazysizes`` and click the 'Activate' button.
 
+You should make sure, that the lazysizes JavaScript is loaded early in order to be able to do it's work as soon as possible. It would not work well if you have the resource loaded at the end of the HTML document.
```

Comment:
I don't understand this statement; do you mind to elaborate?

Reply:
What I mean is, that if the lazyload JS is loaded at the very end of the HTML document, then all images would already been loaded and lazyload could not do it's work.
as long as it's loaded in the HTML head section, all should be good, though. Although I configured the lazyload JS in the project where I use lazyload to be loaded BEFORE the ``plone`` bundle - probably that doesn't have much effect.

Do you have a suggestion for a clearer statement or should I just drop this paragraph?

Reply:
in fact that is not true as the transform is already in place no matter where you place the lazysizes code; just as a test, visit any of these sites with JS disabled:

* http://www.brasil2016.gov.br/pt-br
* https://www.cartacapital.com.br/
* https://www.conversaafiada.com.br/

as you can see no image is loaded at all besides the site logo.

I suggest just to remove this paragraph.

Reply:
You're right. It's the transform which makes the src attr into a data-src something. I should have known. Will remove that paragraph.
... Still it makes sense to load it before any other JS, maybe...?